server/service/src/vaccination/update/validate.rs

```diff
@@ -70,18 +71,40 @@ pub fn validate(
             return Err(UpdateVaccinationError::FacilityNameDoesNotExist);
         }
     }
+
     // If not given, reason is required
     if input.given == Some(false) {
         if input.not_given_reason.is_none() {
             return Err(UpdateVaccinationError::ReasonNotProvided);
         };
     };
 
+    // If selected item is changing - validate it
```

Comment:
I was wondering if there is a description somewhere about what happens in the different use case for immunizations/vaccinations ? (like if i ungive vaccination, if vaccination is given, with stock line issued, and then item is changed etc..)

Reply:
ugh not really... public docs do sort of discuss the different things you can do, but not clear set of use cases like you describe here... where would you write that up??